modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -25,53 +25,69 @@ apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
   name: a-bridge-network
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
 spec:
   config: '{
     "cniVersion": "0.3.1",
-    "name": "cnv-bridge-conf", <2>
+    "name": "cnv-bridge-conf", <1>
```

Comment:
Not sure if it's strictly necessary but this should probably match the name of the NAD (a-bridge-network)
@phoracek will keep me honest.